tests/end-to-end/verify_diff.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+"""Test different output files against expected files."""
+
+import argparse
+
+import sys
+
+import torch
+
+EPS = 0.0001
```

Comment:
since this is global best to add a prefix to it. something like "VERIFY_DIFF_EPS"

Reply:
done.